py_sorter/sorts.py

```diff
@@ -291,3 +291,32 @@ def max_heapify(arr, i, n):
     if largest != i:
         arr[i], arr[largest] = arr[largest], arr[i]
         max_heapify(arr, largest, n)
+
+def shell_sort(integers):
+    """The idea of shellSort is to allow exchange of far items. 
+    An h-sorted array is made for a large value of h. 
+    We keep reducing the value of h until it becomes 1. """
```

Comment:
end of docstrings should be underneath the explanation.

```suggestion
    We keep reducing the value of h until it becomes 1.
    """
```